src/main/groovy/org/jenkinsci/plugins/pipeline/modeldefinition/Utils.groovy

```diff
@@ -151,4 +154,16 @@ public class Utils {
             return false
         }
     }
+
+    @Whitelisted
+    static void setSkippedStage(String stageName) {
+        CpsThread thread = CpsThread.current()
+        CpsFlowExecution execution = thread.execution
+
+        FlowNode currentNode = execution.currentHeads.find { n ->
+            n?.displayName?.equals(stageName)
```

Comment:
Well, irrelevant now anyway since we're just doing an empty stage, but good to know!